File: source/phpfiles/weha/diff/SentenceMove.php
<?php
include_once('SentenceEdit.php');

class SentenceMove extends SentenceMatch {

	public function __construct($oldPos, $newPos, $oldS, $newS, $mr)
	{
		parent::__construct($oldPos, $newPos, $oldS, $newS, $mr);
	}

}

File: google-diff/source/weha/wikianalysis/diff/sentence/SentenceMove.java
package mo.umac.wikianalysis.diff.sentence;

public class SentenceMove extends SentenceMatch {

	public SentenceMove(int oldPos, int newPos, 
						Sentence oldS, Sentence newS, double mr)
	{
		super(oldPos, newPos, oldS, newS, mr);
	}
	
	public SentenceMove(SentenceMatch m)
	{
		super(m);
	}
	
	public String descString()
	{
		return "SentenceMove(" + this.oldPos + ", " + this.newPos + ", " + this.matchingRate + ")";
	}
}
